cyapi/cyapi.py

```diff
@@ -378,7 +378,16 @@ def _generate_urls(self, page_type, page=1, page_size=200, detail="",params={},
                 baseURL = self._add_url_params(baseURL, q_params)
 
             response = self._make_request("get",baseURL)
-            assert response.is_success
+            try:
+                assert response.status_code == 200
+            except AssertionError:
+                print("Failed initial request for {} w/ Status Code: {}".format(page_type, response.status_code))
+                print("get URL:\n{}".format(baseURL))
+                if response.errors:
+                    print("Error(s)")
+                    for k in response.errors:
+                        print("{}: {}".format(k,response.errors.get(k)))
+                exit()
```

Comment:
`exit` is a bit problematic for programs that use this API and want to keep running even when there's an error.
I suggest still raising some kind of error, otherwise this is a breaking change.

Reply:
I appreciate your views on this.

With that line of thinking, I would believe that not only the `exit` would be of concern, but also the `print` statements of the error information.

Would you suggest something other than collecting the currently printed response errors and passing them to:
`raise RuntimeError(errorMessage)`

In place of the `exit()`

Reply:
I completely agree. I think your suggestion for putting everything in the message is great, I can't think of something better.